File: pgm/ABC_Dart/ABC_373_A.dart
// import 'dart:collection';
import 'dart:io';
// import 'dart:math';

void main() {
  const int one = 1;
  const int zero = 0;
  String? line;
  int i = one;
  int cnt = zero;
  while ((line = stdin.readLineSync()) != null) {
    String s = line!;
    if (s.length == i) {
      cnt++;
    }
    i++;
  }
  print(cnt);
  exit(0);
}

File: pgm/ABC_Kotlin/ABC_373_A.kt
// import java.time.*

// import kotlin.math.*

const val one: Int = 1
const val zero: Int = 0

fun main() {
    var line: String? = null
    var i: Int = one
    var cnt: Int = zero
    while (readlnOrNull().also { line = it } != null) {
        var S: String = line!!
        if (S.length == i) {
            cnt++
        }
        i++
    }
    println(cnt)
    kotlin.system.exitProcess(0)
}